schemas/JSON/manifests/v1.2.0/manifest.installer.1.2.0.json

```diff
@@ -550,6 +571,130 @@
         "InstallerUrl",
         "InstallerSha256"
       ]
+    },
+    "NestedInstaller": {
+      "type": "object",
+      "properties": {
+        "InstallerLocale": {
+          "$ref": "#/definitions/Locale"
+        },
+        "Platform": {
+          "$ref": "#/definitions/Platform"
+        },
+        "MinimumOSVersion": {
+          "$ref": "#/definitions/MinimumOSVersion"
+        },
+        "Architecture": {
+          "type": "string",
+          "enum": [
+            "x86",
+            "x64",
+            "arm",
+            "arm64",
+            "neutral"
+          ],

```

Comment:
These should probably be made into definitions

Reply:
Created definition for Architecture